File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Java/Minimum.java
import java.io.*;

class Minimum{
	
	public static void main(String args[])throws IOException{
		
		BufferedReader br  =new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter the number 1 : ");
		int num1 = Integer.parseInt(br.readLine());
		System.out.println("Enter the number 2 : ");
		int num2 = Integer.parseInt(br.readLine());
		System.out.println("Enter the number 3 : ");
		int num3 = Integer.parseInt(br.readLine());

		if(num1<num2 && num1<num3)
			System.out.printf("%d is minimum\n",num1);
		else if(num2<num1 && num2<num3)
			System.out.printf("%d is minimum\n",num2);
		else if(num3<num1 && num3<num2)
			System.out.printf("%d is minimum\n",num3);

	}	

}	

File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Cpp/minimum.cpp
#include<iostream>

int main(){
		
	int num1,num2,num3;

	printf("Enter the number 1 : ");
	scanf("%d",&num1);

	printf("Enter the number 2 : ");
	scanf("%d",&num2);

	printf("Enter the number 3 : ");
	scanf("%d",&num3);

	if(num1<num2 && num1<num3)
		printf("%d is minimum\n",num1);
	else if(num2<num1 && num2<num3)
		printf("%d is minimum\n",num2);
	else if(num3<num1 && num3<num2)
		printf("%d is minimum\n",num3);
}	
